lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -99,8 +99,8 @@
             <span ng-bind="flash.getProgressButtonLabel()"></span>
         </progress-button>
 
-        <p class="step-footer step-footer-split" ng-show="main.state.getFlashState().speed && main.state.getFlashState().percentage != 100">
-          <span ng-bind="main.state.getFlashState().speed.toFixed(2) + ' MB/s'"></span>
+        <p class="step-footer step-footer-split" ng-show="main.state.getFlashState().speed != '0 B/s' && main.state.getFlashState().percentage != 100">
```

Comment:
I think we should still display the speed, even if it's zero, and even if it's at 100%. With compressed images, both cases can happen; i.e. speed will be zero if a lot of blocks are skipped due to blockmapping, or there will still be a non-zero write speed at 100% if a highly compressed image (like bz or gz) is being flashed due to the expansion.

Reply:
We could set speed to an empty string or `'None'` initially and check for that instead